### PR TITLE
Ncl 1881 - UI for alignment reports: show shipped artifacts in a product

### PR DESCRIPTION
**Pull request by vibe13**
@jbartece @alexcreasy @matejonnet Please review. I also added the new config param (DA_REPORT_REST_URL in ui/app/config.json) to the following Jenkins configs:

- http://10.8.49.240:8081/view/Newcastle/job/PNC_MASTER_SNAPSHOT/
- http://10.8.49.240:8081/view/Newcastle/job/PNC_v0.8_DEPLOY/
- http://10.8.49.240:8081/view/Newcastle/job/PNC_Pull_Requests/
- http://10.8.49.240:8081/view/Newcastle/job/PNC_MASTER_1_SNAPSHOT/
- http://10.8.49.240:8081/view/Newcastle/job/PNC_DEMO_DEPLOY/

### CODE REVIEW

**Comment by vibe13** [comment]
Screencast: 
whitelist_report

**Comment by jbartece** [comment]
Looks great! :+1: 
I'll let the merge to @alexcreasy or @matedo1

File: ui/app/common/directives/pnc-select-items.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                return i;
+              }
+            }
+            return -1;
+          };
+
+       	  var PLACEHOLDER = 'Scroll & Filter';
+          $scope.placeholder = _.isUndefined($scope.placeholder) ? PLACEHOLDER : $scope.placeholder;
+          $scope.selectRequired = _.isUndefined($scope.selectRequired) ? false : $scope.selectRequired;
+          $scope.singleItem = convertToBooleanString($scope.singleItem);
```
Comment (matedo1) [inline ui/app/common/directives/pnc-select-items.js]: Hi @vibe13 , why `convertToBooleanString()` needs to be used?

File: ui/app/common/directives/views/pnc-select-items.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <li class="list-group-item" ng-repeat="item in selectedItems">
+        {{ item.fullDisplayText }}
+        <button type="button" class="close" aria-label="Close" ng-click="removeItem(item)">
+            <span aria-hidden="true">×</span>
+        </button>
+    </li>
+</ul>
+<div class="pnc-infinite-select" ng-show="shouldShowSelection()">
+
+    <input ng-required="{{ selectRequired }}" id="{{ selectId }}" name="{{ selectId }}"
```
Comment (matedo1) [inline ui/app/common/directives/views/pnc-select-items.html]: Hi @vibe13 ,  `name="{{ selectId }}"` - there is I think small typo mistake, `name="{{ selectName }}` should be probably used?

**Comment by matedo1** [comment]
Hi @vibe13 , I think nice work! +1 to merge once those minor comments are done

File: ui/app/common/directives/views/pnc-select-items.html (hunk 2 above)
Comment (vibe13) [inline ui/app/common/directives/views/pnc-select-items.html]: @matedo1 yep thanks

File: ui/app/common/directives/pnc-select-items.js (hunk 1 above)
Comment (vibe13) [inline ui/app/common/directives/pnc-select-items.js]: @matedo1 bad naming for the method, was meant to handle proper inittialization of the value, removed to an inline initialization. Thanks!

**Comment by vibe13** [comment]
@matedo1 Thanks Martin, fixed what you mentioned. Rebased against latest master, feel free to press the merge button :-)

**Pull request merged**